packages/plugins/layout/background/src/index.js

```diff
@@ -401,104 +402,105 @@ class PluginComponent extends Component {
               backgroundImage: `linear-gradient(rgba(0, 0, 0, ${darkenFinal}), rgba(0, 0, 0, ${darkenFinal})),linear-gradient(rgba(255, 255, 255, ${lightenFinal}), rgba(255, 255, 255, ${lightenFinal}))`
             }}
           />
-          <BottomToolbar open={focused} theme={darkTheme}>
```

Comment:
I think it would be enough to have `open={focused && !readOnly}` here

Reply:
I'm using the style used elsewhere throughout the codebase (e.g. https://github.com/ory/editor/blob/master/packages/plugins/content/html5-video/src/Component/index.js#L47), which I think makes more sense too. I could wrap it in parenthesis though.

Reply:
Ok, makes sense :)